panel/1.0/webserver_configuration.md

```diff
@@ -109,7 +113,63 @@ sudo a2enmod rewrite
 sudo systemctl restart apache2
 ```
 
+::::
+:::: tab "Caddy With Automatic SSL"
+
+You should paste the contents of the file bellow, replacing `<domain>` with your domain name.
+
+Before adding our custom configuration, let's remove the default one. You can do it either by deleting the contents of config file or by deleting the config file completely and than creating a new one from scratch. The config file path is `/etc/caddy/Caddyfile`.
+
+To delete the config file completely, run the following command:
+
+```shell
+rm /etc/caddy/Caddyfile
+```
```

Comment:
Should be switched around (the text in colors)
![Screenshot_2023-11-29-09-37-32-543_com.android.chrome~2.jpg](https://github.com/pterodactyl/documentation/assets/71085027/97b2e7f0-cbb2-4cd6-98cd-6b6fa625c7c4)